### PR TITLE
Model card and project README

### PR DESCRIPTION
**Pull request by EricMichaelSmith**
Add in MMB model card and project README. TODOs will be fixed before landing.

### CODE REVIEW

File: parlai/zoo/multimodal_blender/README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# Model card: Multi-Modal BlenderBot
+
+Model card for the MMB DegenPos model described in [Multi-Modal Open-Domain Dialogue](TODO: add arXiv link).
+
+## Model details
+This model was trained to create a dialogue agent that can converse engangly about the content of an image as well as about general chitchat. It is a 2.7-billion-parameter Transformer sequence-to-sequence model based on the [BlenderBot](https://ai.facebook.com/blog/state-of-the-art-open-source-chatbot/) open-domain chatbot, trained with image features from [COCO Captions](https://cocodataset.org/) and [Image-Chat](https://parl.ai/projects/image_chat/), and using an image encoder from [Faster R-CNN](https://arxiv.org/abs/1506.01497?context=cs).
```
Comment (klshuster) [inline parlai/zoo/multimodal_blender/README.md:6]: nit: "engagingly"

also, "trained with image features from X datasets" sounds a little off, as the model was trained on those tasks not necessarily just their image features - perhaps "trained on image tasks - [COCO Captions] and [Image-Chat] - using an image encoder from ..."

File: parlai/zoo/multimodal_blender/README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+--n-image-channels 100 \
+--model-file ${DOMAIN_PRETRAINED_MODEL_PATH}
+```
+
+### Fine-tuning
+After the round of domain-adaptive pre-training above, the model must be fine-tuned on the dialogue datasets and on the Image-Chat dataset, using a degendering teacher:
+```
+parlai tm \
+-t genderation_bias:controllable_task:blended_skill_talk,genderation_bias:controllable_task:convai2:normalized,genderation_bias:controllable_task:empathetic_dialogues,genderation_bias:controllable_task:wizard_of_wikipedia,genderation_bias:controllable_task:image_chat:Generation \
+--use-gendered-word-list True \
```
Comment (klshuster) [inline parlai/zoo/multimodal_blender/README.md:68]: remove this flag please (no longer in the teacher)

File: parlai/zoo/multimodal_blender/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    - We find that, in general, “positive” or “neutral” utterances are much less likely to be classified as containing offensive language than “negative” utterances.
+
+Even with these steps, however, we have not addressed all possible safety concerns with this model, and we do not make any kind of guarantee that the model will not produce unsafe or offensive responses. (For instance, even with these measures, 10% of responses given examples from the Image-Chat validation set are still flagged as offensive by the safety classifier presented in [the “Build it break it fix it for dialogue safety” paper of Dinan et al., 2019](https://arxiv.org/abs/1908.06083). However, in this case we have not incorporated the safety features of the `safe_interactive.py` script described above, where the safety classifier mitigates a number of these concerns -- but not all.) If you choose to train or use this model, you do so entirely at your own risk.
```
Comment (klshuster) [inline parlai/zoo/multimodal_blender/README.md:145]: we can probably remove the parentheses around the second/third sentence

**Review by klshuster — APPROVED** [review]
awesome, thanks

File: parlai/zoo/multimodal_blender/README.md (hunk 1 above)
Comment (EricMichaelSmith) [reply parlai/zoo/multimodal_blender/README.md]: yes - changed

File: parlai/zoo/multimodal_blender/README.md (hunk 2 above)
Comment (EricMichaelSmith) [reply parlai/zoo/multimodal_blender/README.md]: cool, removing

File: parlai/zoo/multimodal_blender/README.md (hunk 3 above)
Comment (EricMichaelSmith) [reply parlai/zoo/multimodal_blender/README.md]: Hmm how about keeping it around the 3rd sentence only? That sentence *is* a true parenthetical - I don't want the reader to think that the safety features of `safe_interactive.py` are unusable for our model, for example

Comment (klshuster) [reply parlai/zoo/multimodal_blender/README.md]: actually, after re-reading, happy to keep the full parenthetical. i was missing the forest for the trees

**Pull request merged**